zerver/views/registration.py

```diff
@@ -119,6 +119,7 @@ def accounts_register(request: HttpRequest) -> HttpResponse:
     name_validated = False
     full_name = None
     require_ldap_password = False
+    user_avatar_url = None if not prereg_user.user_avatar_url else prereg_user.user_avatar_url
```

Comment:
Generally we avoid these if/else one-line style, because the normal block style gives us better information about whether both cases have test coverage or not.